internal/master/service.go

```diff
@@ -442,12 +450,44 @@ func (ds *distributedService) ListNodes(ctx context.Context, _ *empty.Empty) (*s
 }
 
 func (ds *distributedService) Close() error {
+	ds.lg.Info("Closing the master service")
 	// Do not invoke DKVService::Close here since `raftRepl` already
 	// closes the underlying storage connection.
+	// TODO - fix the above as ideally it should call DKVService::Close if there are more aspects to DKVService to close other than storage
```

Comment:
Pls check this TODO

Reply:
will have to check raftRepl logic. not a blocker for this PR